### ISSUE TITLE
Option/Command to set a (EX) Raid Category

### ISSUE THREAD

**Issue by mikosoby** [PokeNavBot/issue-tracker#384]
**Is your feature request related to a problem? Please describe.**
It would be nice to have an easy way to set a **(EX) Raid Category**_(the one where the separate Raid channels are created)_.
At the moment if you don't create a category when you set up PokeNav, you have to recreate it later and move everything there which is kinda too much work.

**Describe the solution you'd like**
A simple command which would set a **Raid category** and **EX Raid category**.
Something like `$set raid-category Category Name` and `$set ex-raid-category Category Name`.

**Describe alternatives you've considered**
Option to automatically assing a category according to the placement of the Raid lobby?
But I think that this should might just complicate things a bit and might just be counterproductive. *Pls see below.*
A simple command for this looks as the easiest solution.

**Additional context / What led me to this suggestion**
When we have originally set up the PokeNav bot to our community server we thought, that it would be enough to have everything according to Raids in one category *(which already existed and had permissions assigned and so on)*. But now we see that we've made a bad decision, so last night I created a new EX category, set it up and moved everything there.
As this category is assigned as a **EX Raid Category** I could also configure both Raid Lobbies *(one for regular raids and one for EX ones)* to serve as EX Raid Lobbies... so now it doesn't matter which Lobby is used, the EX Raids will always be created in the assigned category.
But I can't do the same thing for the regular Raids, cause there is no "Regular **Raid Lobby**" and so if I would set the **EX Raid Lobby** to serve also as a regular one, it would create the channels in the EX Raid Category(because of the placement of the channel).
And we have like ~5 other channels(that have nothing to do with PokeNav) in the Raid Category at the moment, so making a new category, moving all the permissions and so on there, set all the channels again is just "not something I would like to do".

**TL;DR** This feature would help servers that didn't thought everything through or just have their community set up and don't want to recreate everything from scratch just for this...
